packages/webhooks/tasks/sendWebhook.spec.js

```diff
@@ -52,6 +55,7 @@ const SendWebhookTests = (appName, actorsInitializer, userCreator, userDestroyer
             const secondUser = await userCreator()
             deletedUser = await userDestroyer(actors.admin, secondUser)
             lastUser = await userCreator()
+            sendWebhook = getWebhookTasks()['sendWebhook']
```

Comment:
Why not just import? To init tasks somewhere else?

Reply:
IDK)